client/src/controllers/MessagesController.ts

```diff
@@ -35,56 +23,99 @@ export class MessagesController extends Controller<HTMLElement> {
   declare readonly templateTargets: HTMLTemplateElement[];
 
   declare showDelayValue: number;
+  declare element: HTMLElement;
```

Comment:
you should not need this, `this.element` will be able to be typed without declaring it this way.